core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheFileInStream.java

```diff
@@ -59,6 +63,9 @@
   private boolean mClosed = false;
   private boolean mEOF = false;
 
+  @VisibleForTesting
+  Ticker mTicker = Ticker.systemTicker();
```

Comment:
for testing purpose, the convention in Alluxio is to still leave this member `private` but introduce a constructor marked ` @VisibleForTesting` to introduce a injected dependency. 
e.g.  
```
public LocalCacheFileInStream(URIStatus status, FileInStreamOpener fileOpener,
      CacheManager cacheManager, AlluxioConfiguration conf) {
  this(status, fileOpener, cacheManager, conf, Ticker.systemTicker());
}


@VisibleForTesting
public LocalCacheFileInStream(URIStatus status, FileInStreamOpener fileOpener,
      CacheManager cacheManager, AlluxioConfiguration conf, Ticker ticker) {
  ...
  mTicker = ticker;
}

```

Reply:
Added a separate constructor for test with stopwatch